R-package/DESCRIPTION

```diff
@@ -7,7 +7,8 @@ Authors@R: c(
     person("Guolin", "Ke", email = "guolin.ke@microsoft.com", role = c("aut", "cre")),
     person("Damien", "Soukhavong", email = "damien.soukhavong@skema.edu", role = c("aut")),
     person("Yachen", "Yan", role = c("ctb")),
-    person("James", "Lamb", email="jaylamb20@gmail.com", role = c("aut"))
+    person("James", "Lamb", email="jaylamb20@gmail.com", role = c("aut")),
+    person("IBM Corporation", role = c("IBM Corporation"))
```

Comment:
`role` should be `ctb` or `aut` I believe.

Reply:
ha of course! I did this too fast, sorry